src/core/qgsvectorlayer.cpp

```diff
@@ -1544,7 +1544,7 @@ bool QgsVectorLayer::readXml( const QDomNode &layer_node, QgsReadWriteContext &c
   }
 
   QgsDataProvider::ProviderOptions options { context.transformContext() };
-  QgsDataProvider::ReadFlags flags = QgsDataProvider::ReadFlags();
+  QgsDataProvider::ReadFlags flags;
```

Comment:
Hi @elpaso, if this line has to be updated, It has to be fixed in other files. Will you do it or do you want that I create a PR just for this ?

This line has been introduced with this PR #38464

Reply:
The generated code is totally equivalent, I just find redundant call the default ctor explicitly. It's more a style matter, nothing important.

Reply:
thanks

Reply:
@rldhont np. 

On another topic I did not check if the same `quotedIdentifier` / `quotedValue` mistake was done elsewhere with your original PR or any followup. Can you please check that it only affected PG?